src/test/javascript/spec/helpers/sample/metis-sample-data.ts

```diff
@@ -20,6 +20,7 @@ export const metisExercise2 = { id: 1, title: 'Second Metis  Exercise', type: Ex
 export const metisUser1 = { id: 1, name: 'username1', login: 'login1', groups: ['metisStudents'] } as User;
 export const metisUser2 = { id: 2, name: 'username2', login: 'login2', groups: ['metisStudents'] } as User;
 export const metisTutor = { id: 3, name: 'username3', login: 'login3', groups: ['metisTutors'] } as User;
+
```

Comment:
Is this change intended?

Reply:
Yes, I want to group objects. It looks better to the eye.